### PR TITLE
Updating packages

### PR DESCRIPTION
**Pull request by iruzevic**
* updating image options component from sidebar to editor
* updated all packages
* small corrections on Storybook
* removing unused packages
* speeding up npm install

### CODE REVIEW

File: blocks/init/src/blocks/components/image/components/image-editor.js [hunk 1]
```diff
@@ -16,6 +23,25 @@ export const ImageEditor = (props) => {
...
+        <MediaPlaceholder
+          icon="format-image"
+          onSelect={onChangeMedia}
+          accept={accept}
+          allowedTypes={allowedTypes}
+        /> :
+        <Fragment>
+          <Button
+            isSecondary
+            onClick={() => onChangeMedia({})}
```
Comment (dingo-d) [inline blocks/init/src/blocks/components/image/components/image-editor.js:37]: Does this have to be a callback? In the MediaPlaceholder you're just calling `onChangeMedia`, but here it's in the arrow function.

File: blocks/init/src/blocks/custom/carousel-image/components/carousel-image-editor.js [hunk 2]
```diff
@@ -7,13 +7,17 @@ export const CarouselImageEditor = (props) => {
...
+    actions: {
+      onChangeMedia,
+    },
   } = props;
 
   return (
     <div className={blockClass}>
       <ImageEditor
         blockClass={blockClass}
         media={(typeof media === 'undefined') || media}
```
Comment (dingo-d) [inline blocks/init/src/blocks/custom/carousel-image/components/carousel-image-editor.js:19]: React handbook mentiones that logic shouldn't be written inline but extracted outside of the JSX block

https://infinum.com/handbook/books/frontend/javascript/react#passing-props

File: blocks/init/src/blocks/custom/image/components/image-editor.js [hunk 3]
```diff
@@ -7,12 +7,16 @@ export const ImageEditor = (props) => {
...
     },
+    actions: {
+      onChangeMedia,
+    },
   } = props;
 
   return (
     <ImageEditorComponent
       blockClass={blockClass}
       media={(typeof media === 'undefined') || media}
```
Comment (dingo-d) [inline blocks/init/src/blocks/custom/image/components/image-editor.js:18]: Inline logic here as well.

File: blocks/init/src/blocks/custom/jumbotron/components/jumbotron-editor.js [hunk 4]
```diff
@@ -29,6 +30,7 @@ export const JumbotronEditor = (props) => {
         <ImageEditor
           blockClass={blockClass}
           media={(typeof media === 'undefined') || media}
```
Comment (dingo-d) [inline blocks/init/src/blocks/custom/jumbotron/components/jumbotron-editor.js:32]: Inline logic

File: blocks/init/src/blocks/components/image/components/image-editor.js [hunk 5]
```diff
@@ -16,6 +23,25 @@ export const ImageEditor = (props) => {
...
-    <img className={imageClass} src={url} alt="" />
+    <Fragment>
+      {!url ?
+        <MediaPlaceholder
+          icon="format-image"
+          onSelect={onChangeMedia}
+          accept={accept}
+          allowedTypes={allowedTypes}
+        /> :
+        <Fragment>
```
Comment (gabriel-glo) [inline blocks/init/src/blocks/components/image/components/image-editor.js:34]: Is the second Fragment necessary?

File: blocks/init/src/blocks/custom/carousel-image/components/carousel-image-editor.js (hunk 2 above)
Comment (IvanGrginovInf) [reply blocks/init/src/blocks/custom/carousel-image/components/carousel-image-editor.js:19]: This is how it's done on all our components if I'm not mistaken (37 results in 31 files).

While you have a valid point, refactoring it only here makes no sense and refactoring it in all files should surely be a separate PR.

Comment (dingo-d) [reply blocks/init/src/blocks/custom/carousel-image/components/carousel-image-editor.js:19]: If it's in that many places I definitely agree, that this should be a separate PR 👍

**Comment by iruzevic** [comment]
No problem I will make a separate PR for this

**Comment by iruzevic** [comment]
I have fixed the inline logic for image. I will send the new PR for the other inline logic fixes. Can you please check this and approve if it is ok?

**Pull request merged**